src/storage/src/hummock/compactor/shared_buffer_compact.rs

```diff
@@ -529,6 +525,13 @@ fn generate_splits(
                 }
             }
         }
+        for table_id in existing_table_ids {
+            if let Some(table_size) = table_size_infos.get(table_id)
+                && *table_size > min_sstable_size
+            {
+                table_vnode_partition.insert(*table_id, 1);
```

Comment:
Is it beneficial to change `shared_buffer_compact` to enable table split? Given that tier-compaction is a must and it will take the whole overlapping level as the input, spliting CN SST seems unnecessary?

Reply:
split to calculate size of each state-table in sst more accurately

Reply:
why do we need to introduce a new config `min_sstable_size` instead of `taget_file_size_base` of `sstable_size` ?

Reply:
Because compute-node can not know `taget_file_size_base `